addons/editor-extra-keys/addon.json

```
@@ -11,25 +11,22 @@
   "info": [
     {
       "type": "notice",
-      "text": "The \"experimental keys\" include equals, slash, semicolon and more. They may not work on all operating systems or keyboard layouts.",
-      "id": "experimentalKeysWarn"
-    },
-    {
-      "type": "notice",
-      "text": "The \"Shift keys\" include keys which typically require the Shift key and a number key, like hashtag, exclamation mark and more. These keys only work with the \"when () key pressed\" block and do not work on all operating systems or keyboard layouts.",
-      "id": "shiftKeysWarn"
+      "text": "Some of these keys may not work on all operating systems or keyboard layouts.",
+      "id": "OSWarn"
     }
   ],
   "settings": [
     {
       "name": "Show experimental keys",
       "id": "experimentalKeys",
+      "description": "Includes keys such as equals, slash, semicolon and others that don't typically require holding shift",
```

```suggestion
      "description": "Includes keys such as equals, slash, semicolon and others that don't typically require holding Shift",
```
We always capitalize key names

---

I was the one who forgot to capitalize it in my suggestion.